px-data-grid-paginated.html

```diff
@@ -79,45 +79,105 @@
 
         static get properties() {
           return {
+            // UNIQUE PROPERTIES FOR PX-DATA-GRID-PAGINATED
+
+            /**
+             * Array of selectable page sizes offered to the user.
+             */
+            selectablePageSizes: {
+              type: Array,
+              value: [10, 20, 30]
+            },
+
+            /**
+             * Current page number of displayed items. Page 1 should be the minimum value.
+             */
+            page: {
+              type: Number,
+              value: 1,
+              observer: '_onPageChange'
+            },
+
+            /**
+             * If true, automatically hides the page numbers list when there is only
+             * 1 page available.
+             */
+            autoHidePageList: {
+              type: Boolean,
+              value: true
```

Comment:
Not related to the docs, but shouldn't we avoid booleans that are true by default?